README.md

```diff
@@ -203,9 +206,69 @@ If you want to use other connectors, you can:
 
 #### Mount a volume containing the plugins
 
-1. Prepare a PersistentVolume which contains a directory with your plugin(s) and their dependencies and ensure 
-   these files are world-readable (`chmod -R a+r /path/to/your/directory`).
-2. Mount the volume into your Pod at the path `/opt/kafka/plugins/`
+You can distribute your plugins to all your cluster nodes into the same path, make them 
+world-readable (`chmod -R a+r /path/to/your/directory`) and use the hostPath volume to mount them into 
+your Kafka Connect deployment. To use the volume, you have to edit the deployment YAML files:
+
+1. Open the [OpenShift template](kafka-connect/resources/openshift-template.yaml) 
+or [Kubernetes deployment](kafka-connect/resources/kafka-connect.yaml)
+2. add the `volumeMounts` and `volumes` sections in the same way as in the example below
+3. Redeploy Kafka Connect for the changes to take effect (If you have Kafka Connect already deployed, you have to apply 
+1. the changes to the deployment and afterwards make sure all pods are restarted. If you haven't yet deployed Kakfa Connect, 
+1. just follow the guide above and use your modified YAML files.)
```

Comment:
the latest "1." create two more bullet points in the list, while the last one should be 3.

Reply:
Fixed